auth/token-managers/vpc-instance-token-manager.ts

```diff
@@ -19,7 +19,7 @@ import { atMostOne } from '../utils';
 import { JwtTokenManager, JwtTokenManagerOptions } from './jwt-token-manager';
 
 const DEFAULT_IMS_ENDPOINT = 'http://169.254.169.254';
-const METADATA_SERVICE_VERSION = '2021-09-20';
+const METADATA_SERVICE_VERSION = '2022-03-01';
```

Comment:
Good question, but the VPC folks said that the reason we are hitting this error is that they started validating/enforcing the "version" param value a few months ago (apparently there hasn't been much use of the VPC authenticators, so we're only seeing this NOW :) ). 

There is really only one version currently ("2022-03-01") so we should be able to continue using this even if/when they deliver a newer version of the API.  The problem occurred because we were using a date value that is before even this first official version.

Reply:
Gotcha, sounds good!